### ISSUE TITLE
Sunlight through the windows ?

### ISSUE THREAD

**Issue by CanKavaloglu** [aalavandhaann/blueprint-js#12]
Hi #OK,

How can I set the windows and/or their materials so that I will have the light passing through the windows just like a sunlight ? Or do I have to use something like Volumetric Light instead of a point light ? 

Thanks in advance !

**Comment by aalavandhaann** [comment]
CanKavaloglu,

This really depends on the models one will use as items in the floorplan. If the items support or have mesh components in them that are transparent or a hole then light will definitely pass through. This is something to work out in the import items.

To simulate sunlight use hemisphere lighting. This [link](https://stackoverflow.com/questions/15478093/realistic-lighting-sunlight-with-three-js) should get you started on sunlight. With regards to volumetric light do you mean something like [this](https://medium.com/@andrew_b_berg/volumetric-light-scattering-in-three-js-6e1850680a41)?

Regards,

#0K

**Comment by CanKavaloglu** [comment]
Hi #OK,

Well, the windows that are already in the project use Mesh Lambert Material and I am setting the opacity to 0 and transparent to true but still no luck. 

By volumetric light I mean something like [this](https://jsfiddle.net/3kkugej8/) and see the [video](https://www.youtube.com/watch?v=suqFV7VGsL4).

Regards

**Comment by CanKavaloglu** [comment]
Hi #OK,

I have fixed the shadows from the windows, it was a matter of configuration in my case. However, the thing is now is that the walls do not receive shadows. Although I changed Mesh Basic Material with Mesh Phong Material in Edge class, the walls do not receive any shadows whereas they cast it. I could not get them to receive the shadows from objects and etc... any suggestions ? 

I believe that this application will be much better with the shadows since they have very good impact on the overall view.

Any help is appreciated in advance.

Cheers !
